src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -817,14 +817,28 @@ end
 settings.set( "shell.allow_startup", true )
 settings.set( "shell.allow_disk_startup", (commands == nil) )
 settings.set( "shell.autocomplete", true )
+settings.set( "shell.prompt_color", "yellow" )
+settings.set( "shell.text_color", "white" )
+settings.set( "shell.background_color", "black" )
 settings.set( "edit.autocomplete", true ) 
 settings.set( "edit.default_extension", "lua" )
+settings.set( "edit.background_color", "black" )
+settings.set( "edit.text_color", "white" )
+settings.set( "edit.highlight_color", "yellow" )
+settings.set( "edit.keyword_color", "yellow" )
+settings.set( "edit.comment_color", "green" )
+settings.set( "edit.string_color", "red" )
 settings.set( "paint.default_extension", "nfp" )
 settings.set( "lua.autocomplete", true )
+settings.set( "lua.prompt_color", "yellow" )
+settings.set( "lua.text_color", "white" )
 settings.set( "list.show_hidden", false )
+settings.set( "list.file_color", "white" )
+settings.set( "list.directory_color", "green" )
```

Comment:
Isn't what exactly same thing as what we achieve by using `colors[ settings.get( "edit.highlight_color" ) ] or colors.yellow`? I think xAnavrins and I are suggesting removing those default from bios.lua and instead documenting them inside help files. There is a fallback build in already into edit.